.golangci.reference.yml

```diff
@@ -1111,6 +1111,22 @@ linters-settings:
     # Default: 1
     tab-width: 1
 
+  loggercheck:
+    # Allow check for the k8s.io/klog/v2 library.
+    # Default: true
+    klog: false
+    # Allow check for the github.com/go-logr/logr library.
+    # Default: true
+    logr: false
+    # Allow check for the "sugar logger" from go.uber.org/zap library.
+    # Default: false
+    zap: true
+    # List of rules to check against, where each rule is a single logger pattern.
+    rules:
```

Comment:
what is the default value? Please add the value to the file.

Reply:
By default the (custom) rules is empty